source/docs/hardware-reference/talonfx/index.rst

```diff
@@ -64,20 +64,210 @@ Status Light Reference
+            </tr>
+            <tr>
+                <td><div class='ledGroup'><div class='led' ontime='300' offtime='300' oncolor='orange' offcolor='black'></div>
+                                          <div class='led' ontime='300' offtime='300' oncolor='orange' offcolor='black'></div></div></td>
+                <td>Blinking Simultaneous Orange</td>
+                <td>Talon FX has valid CAN/PWM signal. If on CAN, Phoenix is running in robot controller <b>and</b> Talon FX has good CAN connection to robot controller.</td>
+            </tr>
+            <tr>
+                <th colspan="3">Enabled Codes</th>
+            </tr>
+            <tr>
+                <td><div class='ledGroup'><div class='led' ontime='0' offtime='0' oncolor='orange' offcolor='orange'></div>
+                                          <div class='led' ontime='0' offtime='0' oncolor='orange' offcolor='orange'></div></div></td>
+                <td>Both Solid Orange</td>
+                <td>Talon FX Enabled with Neutral Output</td>
+            </tr>
+            <tr>
+                <td><div class='ledGroup'><div class='led' ontime='300' offtime='300' oncolor='red' offcolor='black'></div>
+                                          <div class='led' ontime='300' offtime='300' oncolor='red' offcolor='black'></div></div></td>
+                <td>Blinking Simultaneous Red</td>
+                <td>Talon FX driving in reverse. Rate of blink corresponds to duty cycle applied.</td>
+            </tr>
+            <tr>
+                <td><div class='ledGroup'><div class='led' ontime='300' offtime='300' oncolor='green' offcolor='black'></div>
+                                          <div class='led' ontime='300' offtime='300' oncolor='green' offcolor='black'></div></div></td>
+                <td>Blinking Simultaneous Green</td>
+                <td>Talon FX driving forward. Rate of blink corresponds to duty cycle applied.</td>
+            </tr>
+            <tr>
+                <td><div class='ledGroup'><div class='led' ontime='100' offtime='400' oncolor='red' offcolor='black'></div>
+                                          <div class='led' ontime='400' offtime='100' oncolor='black' offcolor='red'></div></div></td>
+                <td>Offset Alternating Red/Off</td>
+                <td>Talon FX limited (hard or soft limit). Direction of offset determines forward/reverse limit.</td>
+            </tr>
+            <tr>
+                <th colspan="3">Special Codes</th>
+            </tr>
+            <tr>
+                <td><div class='ledGroup'><div class='led' ontime='400' offtime='100' oncolor='black' offcolor='orange'></div>
+                                          <div class='led' ontime='100' offtime='400' oncolor='orange' offcolor='black'></div></div></td>
+                <td>Offset Orange/Off</td>
+                <td>Talon FX in thermal cutoff.</td>
```

Comment:
What is thermal cutoff? (what should the customer do?)

Reply:
I'm not sure the blink codes are the place to describe thermal cutoff and solutions to avoid it.

Reply:
I disagree to an extent. It doesn't have to be detailed or fit all cases, just the most common.

As it stands, this information is useless to a customer besides "contact support"... which we should say if we want that to happen